curbs/README.md

```diff
@@ -325,6 +372,96 @@ A Curb Space is represented as a JSON object whose fields are as follows:
+| `max_length` | Integer | Optional | Maximum, bounding box length of the object parallel to the curb, in centimeters. |
+| `max_depth` | Integer | Optional | Maximum, bounding box depth of the object perpendicular to the curb, in centimeters. |
+| `max_height` | Integer | Optional | Maximum, bounding box height of the object from the sidewalk/street surface, in centimeters. |
+| `published_date` | [Timestamp][ts] | Required | The date/time that this curb object was first published in this data feed. |
+| `last_updated_date` | [Timestamp][ts] | Required | The date/time that the properties of ths curb object were last updated. This helps consumers know that some fields may have changed. |
+| `external_object_id` | String | Optional | A unique identifier for this object from an external source, like `external_object_url`. |
+| `external_object_url` | URL | Optional | A URL link to a data feed that contains the `external_object_id`. |
+
+[Top][toc]
+
+### Object Types
+
+The following object types may be specified for Curb Objects. This list is NOT meant to be exhaustive as users have the ability to add objects to this list that may be unique to their city. Descriptions have been provided with each object type where warranted. New object types may be generated to reflect local curb uses, but when possible the following well-known recommended values should be used. If multiple similar values apply, then use the more descriptive/specific value when possible.
+
+**Well-known values:**
+
+Mobility Related
+- `signage` - street sign or any regulation signs related to the curb
+- `bus_stop` - either a sign, shelter, or zone
+- `bike_rack` - somewhere to lock or store a bike
+- `scooter_parking` - dedicated location to lock or store scooters
+- `ev_charging` - charging station for electric devices
+- `ramp` - a curb drop down for accessibility needs
+- `meter` - a device to pay for parking, either single or multi space
+- `pay_station` - a device to pay for parking, applicable for an entire zone
+
+Curbside Infrastructure 
+- `lighting`
+- `signal_cabinet`
+- `utility_box` 
+- `fire_hydrant` 
+- `surveillance_camera`
+
+Curbside Obstacles
+- `barrier`
+- `bollard`
+- `street_trees`
+- `planter`
+- `drinking_fountain`
+- `toilet`
+- `bench`
+- `sculpture`
+- `art`
+- `fountain`
+- `solid_waste_bins`
+- `post_box`
+- `locker`
+
+Other
+- `food_vendor`
+
+[Top][toc]
+
+#### Object Type Attributes
+
+Some object types will have unique, custom attributes that can be optionally included with the given object type. These are given as unordered key-value pairs, e.g. `kilowatts` :  `1.21`. For this current version of Curb Objects, specific object type attributes will not be included in the specification. As use of Curb Objects grows or specific use cases become more involved, common object type attributes can be added on an as needed basis. 
+
+##### Sample Object Type Attributes (for Future)
```

Comment:
This area needs to be fleshed out, and could get complicated fast. It's for custom object fields, so some subset of object types above would need to have their custom fields defined.

Reply:
I think we would be fine to leave this as a list of examples. 
Do we have to tie these attributes to specific object types? Or just leave it open for cities as they see fit.

Reply:
We agreed to remove this entire custom object type attributes section. Make sure references to it are removed from other parts of Objects.

Reply:
Added in latest commit. I also added this for reference

Unique object type attributes for each object type are not included in this version of the specification. As use of Curb Objects grows or specific use cases become more involved, common object type attributes can be added.

Reply:
People on the call agreed that leaving it in would create a breaking change later, and extend the timeline to a future major release (2.0) vs a minor release sooner (1.2). 

It would also create more work now and extend the timeline because for each type of object that already has some type of standard or defintion (EV chargers, trees, parking meters, bike racks, etc) we would have decide on which to use, what to include, and what to modify. 

I think we should leave it out and if people want to create a draft of a version of this they are using, then bring it back to the WG, it will be easier to add later (this is a typical way the spec gets updated).

Reply:
Agreed on leaving it out for now and we can see how Curb Objects is used to add in custom object type attributes later

Reply:
@Mu-yi-Zhou @felixir @mschwartzie To achieve our goal of storing all of the asset data proposed in the SFMTA Internal Curb Data Model (shared by @Mu-yi-Zhou at a previous CDSWG meeting), one option is to do as @schnuerle proposes and implement the custom attribute key value pairs ourselves in our own fork, see how well it works, then come back to the working group.

Reply:
Alright based on conversations with SFMTA and INRIX and OMF, we will leave out custom object attributes for now, and get this PR pushed to the development branch for easier reference and use by all parties. As SFMTA works on implementation, they may bring back other base field suggestions to Objects, and/or a proposal for how to implement custom attributes. If this is done while we are still working on 1.1 in the coming months, we may be able to work it in. If it's done after the 1.1 release approval process is started or complete (likely early Q2 2025) then these ideas could be instead incorporated into a future CDS release.